src/PIL/ImageCms.py

```diff
@@ -819,7 +819,7 @@ def getProfileModel(profile):
         # add an extra newline to preserve pyCMS compatibility
         if not isinstance(profile, ImageCmsProfile):
             profile = ImageCmsProfile(profile)
-        return profile.profile.product_model + "\n"
+        return profile.profile.model + "\n"
```

Comment:
Is there a reason this doesn't also involve the 'or'?
```suggestion
        return (profile.profile.model or "") + "\n"
```

Reply:
Sorry for the late reply, I missed this comment. There probably was a reason but I don't remember now! I've added it for consistency and safety.